Makefile

```diff
@@ -135,13 +135,15 @@ release-test: release-tester ## Test the currently selected release package
 	docker rm -f "$(TESTER_NAME)"
 
 .PHONY: docker-image-build-12 docker-image-build-13 docker-image-build-14
-docker-image-build-12 docker-image-build-13 docker-image-build-14: Dockerfile $(SQL_FILES) $(SRCS) Cargo.toml Cargo.lock $(RUST_SRCS)
+docker-image-build-12 docker-image-build-13 docker-image-build-14: alpine.Dockerfile $(SQL_FILES) $(SRCS) Cargo.toml Cargo.lock $(RUST_SRCS)
 	docker buildx build $(if $(PUSH),--push,--load) \
 		--build-arg TIMESCALEDB_VERSION=$(TIMESCALEDB_VER) \
 		--build-arg PG_VERSION_TAG=$(PG_BUILD_VER) \
+		-t local/dev_promscale_extension:head-ts2-$(PG_BUILD_VER) \
 		-t $(IMAGE_NAME):$(EXT_VERSION)-$(TIMESCALEDB_VER)-$(PG_BUILD_VER) \
 		-t $(IMAGE_NAME):$(EXT_VERSION)-ts$(TIMESCALEDB_MAJOR)-$(PG_BUILD_VER) \
 		-t $(IMAGE_NAME):latest-ts$(TIMESCALEDB_MAJOR)-$(PG_BUILD_VER) \
+        -f alpine.Dockerfile \
```

Comment:
Note: This should transition to the ha.Dockerfile image, _but_ it doesn't build on M1 macs at the moment.

Reply:
I've opened https://github.com/timescale/promscale_extension/issues/140 to track